go/vt/vtgate/planbuilder/operators/delete.go

```diff
@@ -78,12 +76,6 @@ func (d *Delete) GetOrdering(*plancontext.PlanningContext) []OrderBy {
 func (d *Delete) ShortDescription() string {
 	limit := ""
 	orderBy := ""
-	if d.Limit != nil {
-		limit = " " + sqlparser.String(d.Limit)
-	}
-	if len(d.OrderBy) > 0 {
-		orderBy = " " + sqlparser.String(d.OrderBy)
-	}
 
 	return fmt.Sprintf("%s.%s%s%s", d.Target.VTable.Keyspace.Name, d.Target.VTable.Name.String(), orderBy, limit)
```

Comment:
nit: printing of limit and order by should be removed completely. printing empty strings.